.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-    "presets" : ["@babel/preset-env"]
+    "presets" : ["@babel/preset-env"],
+    "env": {
+		"test": {
```

Comment:
These indents are off ? Why?

Reply:
Will edit it.

Reply:
I've been editing and force pushing the indent errors here but its not effecting online. 

I need advice on how to do that.